model/src/test/kotlin/utils/PostgresFileArchiverStorageTest.kt

```diff
@@ -67,7 +72,7 @@ class PostgresFileArchiverStorageTest : WordSpec() {
         }
 
     override fun beforeSpec(spec: Spec) {
-        postgres = EmbeddedPostgres.builder().setPGStartupWait(PG_STARTUP_WAIT).start()
+        postgres = EmbeddedPostgres.builder().setPGStartupWait(PG_STARTUP_WAIT).setPort(port).start()
```

Comment:
I had a look in the code of the `EmbeddedPostgres.Builder` class. IIUC, per default, the builder uses a free random port if no port is explicitly set. (Refer to the `detectPort()` function.) This is actually what we want, so it should not be necessary to set a random port explicitly. You change has the disadvantage that there is a risk that the selected port is already in use.

Reply:
You're right. I admit I didn't dig that deeply and simply added this on spec as I happened to be reminded of the similar code in *HttpFileStorageFunTest*. I'll simply drop the commit.